docs/api/saved-objects/rotate_encryption_key.asciidoc

```diff
@@ -0,0 +1,110 @@
+[role="xpack"]
+[[saved-objects-api-rotate-encryption-key]]
+=== Rotate encryption key API
+++++
+<titleabbrev>Rotate encryption key</titleabbrev>
+++++
+
+experimental[] Rotate encryption key for the saved objects with encrypted attributes.
+
+For the time being if you specify <<xpack-encryptedSavedObjects-keyRotation-decryptionOnlyKeys, decryption-only keys>> {kib} may try to use them all if it cannot decrypt the saved object using the primary encryption key. In most of the cases this overhead is negligible, but if you're constantly dealing with the thousands of the saved objects that are encrypted and experiencing performance issues you may want to try to rotate encryption key for those objects.
```

Comment:
:+1: The funny thing that the Webstorm built-in "spell checker" didn't like `a large number of` when I used initially, maybe it's too casual for the tech docs (I actually like `a large number` better because it's how it would be correct to write in my native language as well) :see_no_evil: 

![1](https://user-images.githubusercontent.com/1713708/96090546-a4704000-0ec8-11eb-85f9-76051078b3e8.png)